tck/build.gradle

```diff
@@ -1,7 +1,7 @@
 description = 'reactive-streams-tck'
 dependencies {
     api group: 'org.testng', name: 'testng', version:'7.3.0'
-    api project(':reactive-streams')
+    api project(path: ':reactive-streams', configuration: 'mrjarArtifact')
```

Comment:
Wires together that the TCK depends on the MRJAR, rather than the unwrapped classes.